pkg/util/utils.go

```diff
@@ -355,10 +354,12 @@ func BuildBasicTags(cluster string, obj interface{}, namespaceID types.UID) []mo
 	switch i := obj.(type) {
 	case *v1alpha1.StaticRoute:
 		tags = append(tags, model.Tag{Scope: String(common.TagScopeNamespace), Tag: String(i.ObjectMeta.Namespace)})
+		tags = append(tags, model.Tag{Scope: String(common.TagScopeNamespaceUID), Tag: String(string(namespaceID))})
```

Comment:
This may cause a different behavior, e.g. the static route will have a empty tag `{"nsx-op/namespace_uid": ""}`, I'm not sure whether it's ok...

Reply:
why we can not get namespaceID from i.ObjectMeta.Namespace instead of passing it from caller?

Reply:
i.ObjectMeta.Namespace is name, not uid